hasher-matcher-actioner/Makefile

```diff
@@ -42,6 +42,21 @@ dev_test_instance:
 	TF_PREFIX=$(call shell-or-die, cd terraform/ &&  terraform output -json prefix) \
 	./tests/scripts/check_deployed_instance.py
 
+dev_upload_test_signal:
+	@TF_BUCKET_NAME=$(call shell-or-die, set -o pipefail && cd terraform/ &&  terraform output -json bucket_name | tr -d '"') && \
+	TF_PDQ_DATA_FILE_KEY=$(call shell-or-die, set -o pipefail && cd terraform/ &&  terraform output -json pdq_data_file_key | tr -d '"') && \
+	echo "Upload sample signals? [y/N] This will copy over $${TF_PDQ_DATA_FILE_KEY} in $${TF_BUCKET_NAME}" && read ans && [ $${ans:-N} = y ] && \
+	aws s3 cp sample_data/pdq.csv s3://$${TF_BUCKET_NAME}/$${TF_PDQ_DATA_FILE_KEY}
```

Comment:
The indexer needs to be able read from multiple pdq files (eg files for 2 collaborations plus one local file). I think ti would be useful to have more descriptive file names so that indexer knows where the index came from. Currently, for pdq hashes from TE were using the following file name format:
```<bucket>/threat_exchange_data/<privacy-group-id>.pdq.te```

Perhaps for local index banks we could use 
```<bucket>/threat_exchange_data/local.<bank-name>.pdq.te```

So this test index could be..

```<bucket>/threat_exchange_data/local.test.pdq.te```

Reply:
This change makes sense eventually but requires changes to terraform and pdq_indexer.py so I will leave it as is for now if that is okay.